network/tests/p2p.rs

```diff
@@ -0,0 +1,73 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0
+
+#[cfg(test)]
+pub mod tests {
+    use std::time::Duration;
+
+    use libp2p::{futures::StreamExt, ping, swarm::SwarmEvent, tcp::TcpConfig, Multiaddr, Swarm};
+    use mina_network::p2p::{MinaTransportBuilder, MinaTransportConfig, MAINNET_CONFIG};
+
+    const DEST: &str =
+        "/ip4/95.217.106.189/tcp/8302/p2p/12D3KooWSxxCtzRLfUzoxgRYW9fTKWPUujdvStuwCPSPUN3629mb";
```

Comment:
where is this from? is it a mina bootnode?

Reply:
This is one of the seed nodes